packages/frontend/src/pages/EmailVerification.tsx

```diff
@@ -0,0 +1,147 @@
+                        just sent to <b>email</b>@email.com
+                    </Subtitle>
+                    <Form
+                        name="verify-email"
+                        onSubmit={onSubmit}
+                        methods={methods}
+                    >
+                        <PasswordInputField
+                            label="One-time password"
+                            name="otp"
+                            placeholder="XXXXXX"
+                            disabled={health.isLoading}
+                            rules={{
+                                required: 'Required field',
+                            }}
+                        />
+                        {nbrOfAttempts < 5 ? (
+                            <>
+                                <Countdown
+                                    date={expirationTime}
+                                    renderer={({
+                                        minutes,
+                                        seconds,
+                                        completed,
+                                    }) => {
+                                        if (completed) {
+                                            setCountdownCompleted(completed);
+                                            return (
+                                                <Subtitle>
+                                                    Your one-time password has
+                                                    expired. <br /> Please
+                                                    resend a verification email.
+                                                </Subtitle>
+                                            );
+                                        }
+                                        return (
+                                            <Subtitle>
+                                                Your one-time password expires
+                                                in{' '}
+                                                <b>
+                                                    {zeroPad(minutes)}:
+                                                    {zeroPad(seconds)}
+                                                </b>
+                                                <br />
+                                                Attempts: {nbrOfAttempts}
+                                                /5
+                                            </Subtitle>
+                                        );
+                                    }}
+                                />
+                                <SubmitButton
+                                    type="submit"
+                                    intent={Intent.PRIMARY}
+                                    text="Submit"
+                                    loading={health.isLoading}
+                                    disabled={countdownCompleted}
```

Comment:
This can just be `completed` from the `renderer` function arguments

Reply:
it's not accessible outside `renderer`